### ISSUE TITLE
matlab library error with spm

### ISSUE THREAD

**Issue by satra** [ReproNim/neurodocker#210]
```
docker run --rm kaczmarj/neurodocker:0.4.1 generate docker     --base debian:stretch \
   --pkg-manager apt --spm12 version=r7219 --vnc passwd=localtest | docker build -t spmvnc -
```

```
# /opt/spm12-r7219/run_spm12.sh /opt/matlabmcr-2010a/v713/ script
My Own Exception: Fatal error loading library /opt/matlabmcr-2010a/v713/bin/glnxa64/libmwmclmcr.so 
Error: libncurses.so.5: cannot open shared object file: No such file or directory
```

**Comment by satra** [comment]
this works for creating a desktop to use with spm batch

```
docker run --rm kaczmarj/neurodocker:0.4.1 generate docker     --base debian:stretch \
  --pkg-manager apt --install libglib2.0-0 libxext6 libsm6 libxrender1 wget libncurses5 \
  --spm12 version=r7219 --vnc passwd=localtest start_at_runtime=True geometry=1920x1080 \
  --run-bash "wget ftp.us.debian.org/debian/pool/main/libx/libxp/libxp6_1.0.2-2_amd64.deb" \
  --run-bash "dpkg -i libxp6_1.0.2-2_amd64.deb" --install xterm openbox | docker build -t spmvnc -
```

**Comment by satra** [comment]
but i don't know which of the installs are essential

**Comment by satra** [comment]
this command installs spm using matlab command runtime and sets up a desktop you can connect to using a vncviewer.

```
docker run --rm kaczmarj/neurodocker:0.4.1 generate docker     --base debian:stretch \
  --pkg-manager apt --install libglib2.0-0 libxext6 libsm6 libxrender1 wget libncurses5 \
  --spm12 version=r7219 --vnc passwd=localtest start_at_runtime=True geometry=1920x1080 \
  --run-bash "wget ftp.us.debian.org/debian/pool/main/libx/libxp/libxp6_1.0.2-2_amd64.deb" \
  --run-bash "dpkg -i libxp6_1.0.2-2_amd64.deb" --install xterm openbox | docker build -t spmvnc -
```

and  then to run spm, run the container

```
docker run --rm -it -p 5901:5901 spmvnc

# /opt/spm12-r7219/run_spm12.sh /opt/matlabmcr-2010a/v713 batch
```
and then from outside connect with a `vncviewer` to `localhost:5901`

**Comment by djarecka** [comment]
@satra - do you know if this is still an issue? should we investigate it?

**Comment by satra** [comment]
i think this may have been partially was addressed in #220. i would close this for now.
